spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/core/GcpContextAutoConfiguration.java

```diff
@@ -72,4 +75,11 @@ public GcpProjectIdProvider gcpProjectIdProvider() {
 
 		return projectIdProvider;
 	}
+
+	@Bean
+	@ConditionalOnMissingBean
+	@ConditionalOnProperty("VCAP_SERVICES")
+	public PcfConfiguration pcfConfiguration() {
```

Comment:
I would recommend calling this CfConfiguration instead. PCF is the Pivotal commercial version of the open source cloud foundry project. It shouldn't matter if you are using PCF or CF (open source), this would work the same way.